packages/apps/library/src/components/LayoutsType/LibraryFont/index.tsx

```diff
@@ -25,17 +25,13 @@ const LibraryFont = (props: any) => {
   return (
     <>
       <Flex flexFlow="row" justify="center">
-        <Tooltip
-          text="Download font here!"
-        >
           <a
             style={{
               fontSize: 12
             }}
             href={link}
-            target="_self"
+            download={true}
```

Comment:
The optional value for download is a string with a file name.
Have you tested just download without the boolean value assigned to it?

Reply:
Hello James, the type of Download is any, I was a bit confused on this topic  but since it was working I thought it was ok.   I removed {true} and I left only download. It's working . Thank you!!